src/application/deployer.go

```diff
@@ -53,5 +54,15 @@ func StartDeploy(deployConfig types.DeployConfig) error {
 	}
 
 	deployConfig.LogChannel <- "Planning deployment..."
-	return terraform.RunPlan(deployConfig, secrets)
+	terraform.RunPlan(deployConfig, secrets)
+	if err != nil {
+		return err
+	}
+
+	if ok := prompt.Confirm("Do you want to continue?"); ok {
```

Comment:
Yes it's prompting to deploy, it's asked after `terraform plan` and before `terraform apply`